susemanager-utils/testing/automation/clean-objects.sh

```diff
@@ -0,0 +1,26 @@
+#/bin/bash -e
+echo "*************** CLEANING OBJECTS GENERATED INSIDE THE CONTAINER  ***************"
+# Exit if the file does not exist or if it is empty
+if [ ! -f /tmp/objects-init.txt -o -s /tmp/objects-init.txt ]; then
+  echo "No file with the initial list of objects"
+  exit 0
+fi
+exit 1
```

Comment:
Exit in both cases? The code below will never be executed.

Reply:
Crap, that was part of a test I did. Good that you reviewed.